components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -465,6 +465,11 @@ abstract class EngineSession(
      */
     abstract fun goForward()
 
+    /**
+     * Navigates to a specific index in the history of this sesson
```

Comment:
Could you add more docs on how this API is supposed to be used to move forward/backward?

Reply:
Also any side-effects if invalid values used.